plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerPlugin.java

```diff
@@ -433,35 +430,6 @@ public void onStop() {
     mapView.removeOnMapChangedListener(onMapChangedListener);
   }
 
-  void onLocationLayerStart() {
```

Comment:
Any reason for changing the access modifiers here? Changing to private causes a synthetic accessor method to be created. In my opinion, it should be the opposite, we should prevent synthetic accessor methods from being created everywhere possible as this adds to the method count.